conf/minion

```diff
@@ -193,6 +193,15 @@
 # The wait-time will be a random number of seconds between 0 and the defined value.
 #random_reauth_delay: 60
 
+
+# To avoid overloading a master when many minions startup at once, a randomized
```

Comment:
Could you also add this documentation to the [minion.rst](https://github.com/saltstack/salt/blob/develop/doc/ref/configuration/minion.rst) page?

It's probably a good idea to add this to the Nitrogen release notes as well.